### ISSUE TITLE
aws_workspaces_workspace can't apply with "ALWAYS_ON" running_mode

### ISSUE THREAD

**Issue by philbal611** [terraform-providers/terraform-provider-aws#13558]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform v0.12.13
+ provider.aws v2.63.0

### Affected Resource(s)

* aws_workspaces_workspace

### Terraform Configuration Files

```hcl
data "aws_workspaces_directory" "directory" {}

resource "aws_workspaces_workspace" "workspace" {
  directory_id = data.aws_workspaces_directory.directory.id
  bundle_id    = "wsb-bh8rsxt14" # Value with Windows 10 (English)
  user_name    = "test.user"

  root_volume_encryption_enabled = true
  user_volume_encryption_enabled = true
  volume_encryption_key          = "aws/workspaces"

  workspace_properties {
    compute_type_name                         = "VALUE"
    root_volume_size_gib                      = 80
    user_volume_size_gib                      = 175
    running_mode                              = "ALWAYS_ON"
    // running_mode_auto_stop_timeout_in_minutes NOT set
  }
}
```

### Expected Behavior

Workspace is deployed with `ALWAYS_ON` running mode.
### Actual Behavior

Workspace fails apply because `running_mode_auto_stop_timeout_in_minutes` is always computed, even when not configured.

```hcl
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_workspaces_workspace.workspace will be created
  + resource "aws_workspaces_workspace" "workspace" {
      + bundle_id                      = "wsb-bh8rsxt14"
      + computer_name                  = (known after apply)
      + directory_id                   = "<directory_id>"
      + id                             = (known after apply)
      + ip_address                     = (known after apply)
      + root_volume_encryption_enabled = true
      + state                          = (known after apply)
      + user_name                      = "test.user"
      + user_volume_encryption_enabled = true
      + volume_encryption_key          = "aws/workspaces"

      + workspace_properties {
          + compute_type_name                         = "VALUE"
          + root_volume_size_gib                      = 80
          + running_mode                              = "ALWAYS_ON"
   ---->  + running_mode_auto_stop_timeout_in_minutes = (known after apply) <----
          + user_volume_size_gib                      = 175
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_workspaces_workspace.workspace: Creating...

Error: workspace creation failed: RunningModeAutoStopTimeoutInMinutes is not applicable for WorkSpace with running mode set to ALWAYS_ON

  on main.tf line 32, in resource "aws_workspaces_workspace" "workspace":
  32: resource "aws_workspaces_workspace" "workspace" {
```
### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by cwebster2** [comment]
I can replicate this bug.  

Using the resource:
```
resource "aws_workspaces_workspace" "example_workspace" {
  directory_id = data.aws_workspaces_directory.example.id
  bundle_id    = data.aws_workspaces_bundle.bundle.id
  user_name    = var.username

  root_volume_encryption_enabled = true
  user_volume_encryption_enabled = true

  workspace_properties {
    compute_type_name                         = var.compute_type
    user_volume_size_gib                      = 50
    root_volume_size_gib                      = 80
    running_mode                              = "ALWAYS_ON"
  }
}
```

Results in the error
`Error: workspace creation failed: RunningModeAutoStopTimeoutInMinutes is not applicable for WorkSpace with running mode set to ALWAYS_ON
` during apply after the plan decides to add `running_mode_auto_stop_timeout_in_minutes` to my resource in error.

**Comment by anGie44** [comment]
The fix for this bug has been merged and will release with `v3.5.0` of the Terraform AWS Provider, likely out this Thursday.
